connectors/salesforce/src/main/java/com/vmware/connectors/salesforce/SalesforceController.java

```diff
@@ -111,252 +117,281 @@ public SalesforceController(AsyncRestOperations rest, CardTextAccessor cardTextA
         this.sfAttachmentTasklinkPath = sfAttachmentTasklinkPath;
     }
 
-    @PostMapping(path = "/cards/requests", consumes = APPLICATION_JSON_VALUE, produces = APPLICATION_JSON_VALUE)
+    ///////////////////////////////////////////////////////////////////
+    // Methods common to both the cards request and the actions
+    ///////////////////////////////////////////////////////////////////
+
+    private HttpHeaders makeHeaders(String auth) {
```

Comment:
Most of the methods in this file flow top-down (if A calls into B and C, and B calls into D, and D calls into E, then the order of my functions in source would be: A, B, D, E, C -- depth first traversal).  These are the few methods that are used in more than 1 place (and are relatively small and few in number), so I put them at the top instead.